doc/object_properties.adoc

```diff
@@ -0,0 +1,252 @@
+= Object Properties
+include::../include/config.adoc[]
+:url-c-content-src: {url-engine-src}/script/common/c_content.cpp
+
+WARNING: There may be missing/incorrect information here,
+please refer to the {url-lua-api}#L7141-L7304[Lua API]
+and the {url-c-content-src}#L186-L442[relevant engine code]
+
+WARNING: This is missing the default values for everything
+
+== Properties Applying Only to Players
+
+=== `hp_max`
+* internal type: `u16`
+
+Sets the max HP of a player.
+Clamped if it goes above the `u16` max value.
+
+=== `breath_max`
+* internal type: `u16`
+
+Sets the max breath of a player.
+
+=== `eye_height`
+* internal type: `f32`
+
+How high the camera is in nodes about the player's feet.
+
+=== `zoom_fov`
+* units: `degrees`
+
+Changes the field of view of player when they zoom. Setting to `0` disables zooming.
+
+== Properties Applying to All Objects
+
+=== `physical`
+* field type: `bool`
+
+Collides with nodes that are `walkable`.
+
+=== `collide_with_objects`
+* field type: `bool`
+
+Collides with other objects.
+CAUTION: `physical` must be set to `true` for this to take effect!
```

Comment:
You need a newline before this, or it interprets it as part of the paragraph, so it doesn't display correctly.